examples/gpx.html

```diff
@@ -46,15 +43,65 @@
             itowns.Fetcher.json('./layers/JSONLayers/WORLD_DTM.json').then(addElevationLayerFromConfig);
             itowns.Fetcher.json('./layers/JSONLayers/IGN_MNT_HIGHRES.json').then(addElevationLayerFromConfig);
 
+            // update the waypoint
+            var point = new itowns.THREE.Vector3();
+            function getDistance(object, camera) {
+                point.copy(object.geometry.boundingSphere.center);
```

Comment:
if there's not matrix transform, so point is useless.
You could write : 
```js
return camera.position.distanceTo(object.geometry.boundingSphere.cente);
```

Reply:
Done